src/libraries/Microsoft.PowerFx.Core/Binding/BinderUtils.cs

```diff
@@ -353,18 +358,21 @@ private static BinderCheckTypeResult CheckTypeCore(IErrorContainer errorContaine
                 return new BinderCheckTypeResult() { Coercions = coercions };
             }
 
-            // If the node is a control, we may be able to coerce its primary output property
-            // to the desired type, and in the process support simplified syntax such as: label1 + slider4
-            IExternalControlProperty primaryOutProp;
-            if (nodeType is IExternalControlType controlType && node.AsFirstName() != null && (primaryOutProp = controlType.ControlTemplate.PrimaryOutputProperty) != null)
+            if (!features.PrimaryOutputPropertyCoercionDeprecated)
```

Comment:
>PrimaryOutputPropertyCoercionDeprecated [](http://example.com/codeflow?start=26&length=39)

Is this (and the one one line 296) the only significative change in the code (besides passing features instead of just PFxV1)?

Reply:
yup.